helpers

```diff
@@ -69,3 +81,37 @@ get_udevadm_info_dev ()
 }
 export -f get_udevadm_info_dev
 
+# Converts the given seconds ($1) to DDd:HHh:MMh:SSs format
+seconds_to_date ()
+{
+    local seconds="$1"
+    local days=$((seconds/86400))
+    local hours=$((seconds/3600%24))
+    local mins=$((seconds/60%60))
+    local secs=$((seconds%60))
+
+    printf '%02dd:%02dh:%02dm:%02ds' $days $hours $mins $secs
+}
+export -f seconds_to_date
+
+get_sosreport_time ()
+{
+    local sos_path="${DATA_ROOT}/sos_commands/date/date"
+    if [ -s "$sos_path" ]; then
+        date --date="$(cat $sos_path)" +%s
+    elif ! [ -d "${DATA_ROOT}sos_commands" ]; then
+        date +%s
+    fi
+}
+export -f get_sosreport_time
+
+get_ceph_volume ()
```

Comment:
can we call this get_ceph_volume_lvm_list() just in case we want to add other ceph-volume commands